pennylane/transforms/optimization/cancel_inverses.py

```diff
@@ -165,9 +168,8 @@ def qfunc(x, y, z):
         # - there is no wire symmetry
         # - the control wire symmetry does not apply because the control wires are not the same
         # - neither of the flags are_self_inverses and are_inverses are true
-        apply(current_gate)
+        operations.append(current_gate)
         continue
 
-    # Queue the measurements normally
-    for m in tape.measurements:
-        apply(m)
+    new_tape = QuantumTape(operations, tape.measurements, shots=tape.shots)
+    return [new_tape], lambda x: x[0]
```

Comment:
Even in cases with the "null processing functions", let's try to have a fully named and documented function, 
```
def null_postprocessing(results):
    """A postprocesing function returned by a transform that only converts the batch of results
    into a result for a single ``QuantumTape``.
    """
    return results[0]

```
```
    ....
    return (new_tape, ), null_postprocessing
```

This will make logging and debugging substantially easier in the long term, and only costs a couple of extra lines.